x-pack/plugin/ml/qa/ml-with-security/src/test/java/org/elasticsearch/smoketest/MlWithSecurityInsufficientRoleIT.java

```diff
@@ -60,5 +80,27 @@ public void test() throws IOException {
     protected String[] getCredentials() {
         return new String[]{"no_ml", "x-pack-test-password"};
     }
+
+    @SuppressWarnings("unchecked")
+    static boolean containsKey(String key, Map<String, Object> mapOfMaps) {
+        Set<String> keys = mapOfMaps.keySet();
+        for (String keyEntry : keys) {
```

Comment:
Isn't this the same as doing `if (mapOfMaps.contains(key)) return true;` ?

Reply:
Ha yeah that might be a simpler way of doing it